readthedocs/projects/views/private.py

```diff
@@ -1206,3 +1207,18 @@ def _get_csv_data(self):
 
     def _get_feature(self, project):
         return get_feature(project, feature_type=self.feature_type)
+
+
+class ProjectPullRequestsUpdate(PrivateViewMixin, UpdateView):
+    model = Project
+    form_class = ProjectPullRequestForm
+    success_message = _("Pull request settings have been updated")
+    template_name = "projects/project_pull_requests.html"
```

Comment:
In the new dashboard, I'm trying to stick to a naming convention closer to Django's built in naming and I have been removing our prefixes like `project_`. For this, I'd use:

```suggestion
    template_name = "projects/pull_requests_form.html"
```